packit.spec

```diff
@@ -0,0 +1,60 @@
+%global pypi_name packitos
+%global real_name packit
+
+Name:           %{real_name}
+Version:        0.0.1
+Release:        1%{?dist}
+Summary:        A set of tools to integrate upstream open source projects into Fedora operating system
+
+License:        MIT
+URL:            https://github.com/packit-service/packit
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(setuptools)
+BuildRequires:  python3dist(setuptools-scm)
+BuildRequires:  python3dist(setuptools-scm-git-archive)
+Requires:       fedpkg
+Requires:       python3-%{real_name}
+
+%?python_enable_dependency_generator
+
+%description
+This project provides tooling and automation to integrate upstream open source
+projects into Fedora operating system.
+
+%package -n     python3-%{real_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{real_name}}
+
+%description -n python3-%{real_name}
+This project provides tooling and automation to integrate upstream open source
+projects into Fedora operating system.
+
+This package contains the python code,
+check out packit package for the executable.
+
+
+%prep
+%autosetup -n %{pypi_name}-%{version}
```

Comment:
Instead of `pypi_name` use `real_name` otherwise it fails with 
~~~
BUILDSTDERR: /var/tmp/rpm-tmp.zWis4U: line 38: cd: packitos-0.0.1: No such file or directory
~~~